bedrock/contentful/templates/includes/contentful/cta.html

```diff
@@ -10,8 +10,7 @@
 
 {% if action == 'Download Firefox' %}
 
-  <!-- TODO: unique dom_ID -->
-  {{ download_firefox_thanks(dom_id='download-button-primary', download_location=location, alt_copy=label, button_class=button_class) }}
+  <a href="{{ url('firefox.browsers.index') }}" class="mzp-c-button {{ button_class }}" data-cta-text="{{ cta_text }}" data-cta-type="link" data-cta-position="{{ location }}">{{ label }}</a>
```

Comment:
As discussed on Slack (also not a blocker), it feels a little odd to me for the action `Download Firefox` not to render a download button any longer. Perhaps this could be a new action, like `Explore Firefox`? I think we should probably keep the ability to render a real download button for pages that need it.